### DISCUSSION TITLE
Why All the Component passthrough is missing, i cant find it from official website.

### DISCUSSION THREAD

**Discussion by dugufu** [primefaces/community#230] [PrimeVue]
![image](https://github.com/primefaces/community/assets/49250073/8162372a-b51e-4a15-baa8-3599194ea885)
![image](https://github.com/primefaces/community/assets/49250073/d5432a2a-acf8-4c7e-ae84-9cd008503913)

It only show wireframe and demo. This is from official website
https://primevue.org

**Comment by BASEFlyer** [comment]
Good question! I was about to ask it.

**Comment by BASEFlyer** [comment]
For right now im using Theming to find PT options
![image](https://github.com/primefaces/community/assets/118068574/d87d741f-496d-4385-80e6-f27acfddd425)
its pain but better then nothing.

**Comment by BASEFlyer** [comment]
> ![image](https://user-images.githubusercontent.com/49250073/274915309-8162372a-b51e-4a15-baa8-3599194ea885.png) ![image](https://user-images.githubusercontent.com/49250073/274915567-d5432a2a-acf8-4c7e-ae84-9cd008503913.png)
> 
> It only show wireframe and demo. This is from official website https://primevue.org

Also you can find all options under API tab

**Comment by FlipWarthog** [comment]
This issue has been fixed.

**Comment by tugcekucukoglu** [comment]
Live demos are updated.

Thank you!
